src/NServiceBus.TransportTests/NServiceBusTransportTest.cs

```diff
@@ -139,15 +139,13 @@ protected Task SendMessage(string address, Dictionary<string, string> headers =
 
             var dispatcher = lazyDispatcher.Value;
 
-            var context = new ContextBag();
-
             //until we fix the seam we have to do this
             if (transportTransaction != null)
```

Comment:
Pretty sure the logic here is now wrong, since we'd only want to create a new `TransportTransaction` if we didn't get one passed in, right?